docs/guides/continuous-integration/aws-codebuild.mdx

```diff
@@ -186,8 +186,7 @@ batch:
   build-list:
     - identifier: cypress-e2e-tests
       env:
-        variables:
-          IMAGE: public.ecr.aws/cypress-io/cypress/browsers:node12.14.1-chrome85-ff81
+        image: public.ecr.aws/cypress-io/cypress/browsers:node12.14.1-chrome85-ff81
```

Comment:
Perhaps this is also a good opportunity to update to a Docker image with a supported version of Node.js i.e. `16` or later? See https://github.com/nodejs/release#release-schedule.

Also the version could be aligned with the RWA example in
https://github.com/cypress-io/cypress-realworld-app/blob/develop/buildspec.yml